src/AutoPilotPlugins/PX4/RadioComponent.cc

```diff
@@ -140,7 +140,11 @@ QStringList RadioComponent::paramFilterList(void) const
 
 QWidget* RadioComponent::setupWidget(void) const
 {
-    return new PX4RCCalibration;
+    QGCQmlWidgetHolder* holder = new QGCQmlWidgetHolder();
```

Comment:
> Only one thing, seems that compiling is slower, over one minute for such a small project.

I haven't noticed much compile time increase. TeamCity is reporting the same builds time for the last couple weeks.